packages/adapter-sveltekit/src/hooks/project-init.ts

```diff
@@ -55,7 +55,7 @@ const createPrismicIOFile = async ({
 	if (isTypeScriptProject) {
 		contents = source`
 			import * as prismic from "@prismicio/client";
-			import { CreateClientConfig, enableAutoPreviews } from '@prismicio/svelte/kit';
+			import { type CreateClientConfig, enableAutoPreviews } from '@prismicio/svelte/kit';
```

Comment:
Is it required now, or it's just cleaner code?

Reply:
Required now. SvelteKit 2 adds `verbatimModuleSyntax` to the generated `tsconfig.json` file, which requires declaring `type` when importing types.

Reply:
(Good question!)